charts/argo-templates/scripts/lookup-file-id.py

```diff
@@ -11,14 +11,16 @@
 
 authed_session = AuthorizedSession(credentials)
 
-def check_file_existence(target_path: str):
+def get_file_id(target_path: str):
     response = authed_session.get(f"{base_url}/api/repository/v1/datasets/{dataset_id}/filesystem/objects",
                                   params={"path": target_path})
     if response.status_code == 200:
-        return "true"
-    elif response.status_code == 404:
-        return "false"
-    else:
+        return response.json()["fileId"]
+    elif response.status_code != 404:
```

Comment:
what happens if it is a 404?

Reply:
(I think) the function will "return" `None`

Reply:
this is a pythonism that I don't like haha, apparently you can just leave off an else. 🤷 fine by me then